### ISSUE TITLE
Events are no longer logged to VoiceAttack's event log and VoiceAttack actions for reacting to events do not trigger

### ISSUE THREAD

**Issue by Tkael** [EDCD/EDDI#395]
## EDDI version in which issue found
[3.0.0-b2]

## Steps to reproduce
As described here: https://forums.frontier.co.uk/showthread.php/387955-EDDI-2-4-Bring-your-cockpit-to-life?p=6461680&viewfull=1#post6461680

## Expected
Events should be logged to VoiceAttack's event log and VoiceAttack actions for reacting to events should trigger

## Observed
Events are no longer logged to VoiceAttack's event log and VoiceAttack actions for reacting to events do not trigger

## Investigation
This may be a bug introduced by the changes introduced to create a single instance of EDDI (whether standalone or via VoiceAttack)?

**Comment by Tkael** [comment]
This is potentially a big issue - it looks like it breaks every VoiceAttack command that relies on a trigger from EDDI to run.

**Comment by Tkael** [comment]
Ok. From Git bisect, I think I've found the offending commit:
```
3be0accace1e00fbe347e8a5bb4ed98bfacada86 is the first bad commit
commit 3be0accace1e00fbe347e8a5bb4ed98bfacada86
Author: Tkael <dragoncharmer84@hotmail.com>
Date:   Sun Feb 4 20:36:51 2018 -0800

    Add support for status.json

    Many new variables are now available under the `status` object (see updated                                                 Variables.md).

    Added the following new uder-facing events:
    - `Near surface` event
    - `SRV under ship` event
    - `SRV turret` event
    - `Ship fsd` event
    - `Ship low fuel` event

    Note: The new status object is also reflected to VoiceAttack. If this PR is                                                 approved, the documentation for VoiceAttack will need to be updated per the new                                                 values available in the VoiceAttackPlugin.cs `setStatusValues()` method.

:040000 040000 1ed6ec51d2f11d182de0ace5bcdf2cb97ef5af38 2568512c4db474cbd3ede3b1                                                c05d87e698d541e7 M      DataDefinitions
:100644 100644 93429f788d2666c24dfc59f8f960892b568a56ba aaac82ff34f2683da022e417                                                aac6892f898d66a2 M      EDDI.sln
:040000 040000 f21f0d338928d482d89a7ed35dbf707184aee286 ff4c1da3a20080562506bc62                                                ac086a5c8ce0edd9 M      EDDI
:040000 040000 111f5f1f5bf7805534119add5a1ffa82eceaae5e af67374c2733b5d64bed76d5                                                12c6222bdbef3cb4 M      Events
:100644 100644 fb3810ec95e4c67c847a97a828294f63f414c33f 904fbfbc0945feae69a4b24e                                                b3ef678aed48c0bd M      Installer.iss
:040000 040000 4e1c06d80a3b780d55bc6286fdf7706b8aa7dabb b291007d478133d9804cfcd2                                                c6a19e17ce95362c M      ShipMonitor
:040000 040000 bb41e0f179a9e63febc1154d672e8dd56adc81bc 3cb94cadf361faabf24ca09f                                                c8ba383014cffb77 M      SpeechResponder
:000000 040000 0000000000000000000000000000000000000000 3507ef45769e02bd1beeef57                                                538b773b1dd862ff A      StatusMonitor
:040000 040000 bac4638dbb82cd56d353c607db8193d3c279f6f7 9168863e421eda10595b1837                                                e76aeb4e6a5610ba M      VoiceAttackResponder
```
Looks like my changes for the new status.json file broke it. Now to find out how I broke it...

**Comment by richardbuckle** [comment]
Not at my PC right now, but on a cursory review on my iPad I noticed some usage of global variables. That’s generally a bad mix with multi-threading and is probably where I’d look first.

**Comment by Tkael** [comment]
Closed pending approval of #403
